Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -260,10 +260,12 @@ class TouchableOpacity extends React.Component<Props, State> {
   }
 
   componentDidUpdate(prevProps: Props, prevState: State) {
+    //function to check equality of two objects
+
```

Comment:
I think this comment is no longer necessary 

```suggestion 
```

Reply:
@gabrieldonadel Done! Anything else I need to do?